patch_scene.py

```diff
@@ -10,6 +11,13 @@
 from src.utils.nuscenes_helper import group_instances_across_frames
 
 
+def parse_config():
+    parser = argparse.ArgumentParser(description='arg parser')
+    parser.add_argument('--start_scene_index', type=int, default=0, help='specify your scene index to start with')
```

Comment:
we need to check if `NuScene` loads scenes in a consistent ways across different launches: if this statement does not hold then `start_scene_index` may reflect **previous** ordering which is changed on a new run.